src/components/Map.test.js

```diff
@@ -1,37 +1,110 @@
 import React from 'react';
 import { render, cleanup } from '@testing-library/react';
 import { Map } from './Map';
+import { NEW_LAYER, RENDERING_POLYLINE, RENDERING_HEATMAP, RENDERING_CLUSTERS } from './Layer';
 
-// TODO: Define specs and add unit tests
-// TODO: Consider moving mock store setup to a util
-describe('Map tests', () => {
-  it('renders the map', () => {
-    const map = render(<Map />).container;
-    expect(map).toBeDefined();
-  });
+jest.mock('leaflet');
+
+// TODO: Complete test cases!!!
```

Comment:
@stellasia, @norbertFeron, @wagnerjt turns out unit testing map behavior with leaflet rendering is challenging. With some research, I've seen what libs like react-leaflet do to validate map, and I actually copied react-leaflet [jest mock of leafletjs](../../__mock__/leaflet.js). I believe that's half the puzzle. For now, these UTs remain placeholders and I reverted to manual testing as set up in this [gist](https://gist.github.com/jodom/81db1914dffad68cd0b7a6b3c4d2de2b). Behavior proposed in this PR check out with no regressions.

This also makes me think that we need two things:
1. A manual validation set up for the development workflow. This should work with a mock neomap driver, allowing for full simulation of connection to graph
2. Integration / e2e tests to augment the UTs we are currently building. While manually validating the map, I see bugs in the **SideBar** and **Layer** components. While these may be buggy because we don't yet have sufficient coverage, I think we always will need e2e tests for things UTs wont catch.

Publishing this PR as ready to merge. Let me know the experience you have from [manual validations](https://gist.github.com/jodom/81db1914dffad68cd0b7a6b3c4d2de2b).

Deferring UT work here until I have more time